bluemira/equilibria/optimisation/harmonics/harmonics_approx_functions.py

```diff
@@ -137,8 +109,7 @@ def coil_harmonic_amplitude_matrix(
         * lpmv(ones, degrees, np.cos(theta_f)[None, :])
         / np.sqrt(degrees * (degrees + 1))
     )
-
-    return sig_fig_round(currents2harmonics, 8)
+    return sig_fig_round(currents2harmonics, 15)
```

Comment:
I think these 15's should be passed in as params and be defaulted to 15

Reply:
Hey, I have no problem with making it a default rather than hard coding it in. I have done that and it is in the latest commit.